src/containers/group-management/group-detail-role-management/preview-roles.tsx

```diff
@@ -0,0 +1,64 @@
+import { Trans } from '@lingui/macro';
+import React from 'react';
+import { Flex, FlexItem, Label, Divider } from '@patternfly/react-core';
+
+import { RoleType, GroupObjectPermissionType } from 'src/api';
+
+import { Tooltip } from 'src/components';
+
+import { Constants } from 'src/constants';
+
+import './group-detail-role-management.scss';
+
+interface Props {
+  group: GroupObjectPermissionType;
+  selectedRoles: RoleType[];
+}
+
+const splitByDot = (perm: string) => {
+  const [category, permission] = perm.split('.');
```

Comment:
```suggestion
  const [category, permission] = perm.split('.', 2);
```

(if the string is `foo.a.b.c`, you want permission to be `a.b.c`, not just `a`, right?)